inference-engine/ie_bridges/python/requirements.txt

```diff
@@ -1 +1 @@
-numpy~=1.19.5
\ No newline at end of file
+numpy>=1.16.6,<1.20
```

Comment:
Hi! Why we limited by NumPY 1.20? With this version Python API works fine for me,

Reply:
see https://github.com/openvinotoolkit/openvino/pull/8653